### ISSUE TITLE
conf/mysql is missing

### ISSUE THREAD

**Issue by pquerner** [JeroenBoersma/docker-compose-development#56]
Link: https://github.com/JeroenBoersma/docker-compose-development/blob/master/conf/mysql (from the README)
Should be added here https://github.com/JeroenBoersma/docker-compose-development/commit/8955a1389438687dedf91000e93b933ea818aa15
but is missing anyway.

**Comment by Caroga** [comment]
Glad you mentioned it. It was an idea to include this but this actually broke existing installations so it has been removed. Should update the readme on this. 
You can just create your own conf/mysql with the following content:
```
MYSQL_ROOT_PASSWORD=<yourpassword>
```

**Comment by pquerner** [comment]
A conf/mysql.dist file could be shipped, and if the users trys to "start" this app (dev up), and the "conf/msql" file is missing, the user could be prompted for a password and the file "conf/mysql" could be created for him/her.

**Comment by Caroga** [comment]
This is being implemented in https://github.com/JeroenBoersma/docker-compose-development/issues/51
For now, this issue can be closed.

**Comment by pquerner** [comment]
Great! Should this issue not be closed before the README has been adapted?

**Comment by Caroga** [comment]
Sure, I will probably PR this tonight along with some other outstanding remarks. Will reference this issue so it can be closed upon merging.

**Comment by JeroenBoersma** [comment]
Hi all, thanks for bringing this up, as mentioned before when you would pull into an existing environment and you use a secure/string root password then it would be overwritten and you'll need to restart or start mysql  into a no privileges mode to reset the root password.

Looking forward to your PR @caroga, please mention this ticket and than we'll close it

**Comment by pquerner** [comment]
The file is once again missing.

**Comment by JeroenBoersma** [comment]
@pquerner ah, thanks... Moved it to `.env` docs are running behind... again, will update tonight...
I created fallback compatible logic, if the `conf/mysql` file already exists, it'll use that one, if it doesn't it wants to take the password from `.env`

I've added `bin/dev setup` which will create a random root password and creates a user same as your system user with access to `{USERNAME}_*` databases.
This is a security thing that your databases are prefixed locally to make sure you are not accidentally removing a production environment.

`dev myroot` will automaticly log you in as `root` on MySQL and `dev mysql` will log you in as you without a password.

Hope you'll give it a try!
